src/GT.UI/Controllers/API/UserTestController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using GT.UI.Models;
```

Comment:
Same as admin controller comment.

Reply:
We could leave it out from branch, or simply delete these that were used for test. Included them in case #58 could draw benefit from these files.

Reply:
Good to keep in the branch for testing purposes, but should be left out of main. Could we perhaps branch the branch?

Reply:
Removed from PR.